core_backend/app/users/schemas.py

```diff
@@ -0,0 +1,17 @@
+from datetime import datetime
+
+from pydantic import BaseModel, ConfigDict
+
+
+class UserCreate(BaseModel):
+    """
+    Pydantic model for user creation
+    """
+
+    username: str
+    user_id: str
+    # retrieval_token: str
+    created_datetime_utc: datetime
+    updated_datetime_utc: datetime
```

Comment:
Same here. Probably scrap this field.

Reply:
Removed in PR #179. This should always be handled by the DB-insert function anyway.